PackageViewModel/PackageViewModel.cs

```diff
@@ -83,7 +85,8 @@ public sealed class PackageViewModel : ViewModelBase, IDisposable
 
             _packageMetadata = new EditablePackageMetadata(_package, UIServices);
 
-            _packageSource = source;
```

Comment:
@onovotny I noticed that you recently changed this to set the field instead of the property. Was this intended? By setting the property a `FileSystemWatcher` would have been created to watch changes to the file.

Reply:
That was to make a C# 8 warning go away. I didn’t realize the property did more, it should set the property then.

Reply:
Ah, yeah I currently also get the warning. Should I just make the fields nullable then?

Reply:
No, it can be suppressed either with the ! post-fix operator or a pragma.